ui/lib/core/addon/components/form-field.js

```diff
@@ -31,7 +31,8 @@ import layout from '../templates/components/form-field';
 export default Component.extend({
   layout,
   'data-test-field': true,
-  classNames: ['field'],
+  attributeBindings: ['data-test-field'],
+  classNames: ['field-foo'],
```

Comment:
Is this intentional, to keep `field-foo`? or is this a placeholder for something to be overwritten?

Reply:
Hmm that can't be right. Maybe a hold over from some testing but the class should be `field`. Good catch!